src/types/BaseField.ts

```diff
@@ -55,6 +56,10 @@ const baseFieldSchema: JSONSchemaType<BaseField> = {
 			type: 'string',
 			enum: Object.values(BaseFieldScope),
 		},
+		relevanceDuration: {
+			type: 'number',
```

Comment:
Regarding my wistful wish for duration types: ah, but JSON and OpenAPI and AJV. Sigh.

Perhaps to make it more explicit that the unit is seconds, can we include the unit in the name? E.g. `valueRelevanceSeconds`?